cli/flox/doc/doc-compose/flox-list.md

```diff
@@ -0,0 +1,49 @@
+---
+title: FLOX-LIST
+section: 1
+header: "Flox User Manuals"
+...
+
+
+# NAME
+
+flox-list - list packages installed in an environment
+
+# SYNOPSIS
+
+```
+flox [<general-options>] list
+     [-d=<path> | -r=<owner/name>]
+     [-e | -c | -n | -a]
+```
+
+# DESCRIPTION
+
+List packages installed in an environment.
+The options `-n`, `-e`, and `-a` exist to provide varying levels of detail in
+the output.
+
+# OPTIONS
+
+## List Options
+
+`-e`, `--extended`
+:   Show the install ID, pkg-path, and version of each package (default).
+
+`-c`, `--config`
+:   Show the raw contents of the manifest. When using composition, this will be
```

Comment:
nit: newline after sentences

Reply:
We try to follow sembr for docs. We have that written down for floxdocs, not sure if we need to document that in flox/flox https://github.com/flox/floxdocs/blob/preview/README.md?plain=1#L66